server/package.json

```diff
@@ -70,5 +74,6 @@
     "mongodbMemoryServer": {
       "debug": "1"
     }
-  }
+  },
+  "type": "module"
```

Comment:
This changes this project from CJS to ESM. Is that supposed be part of the scope of this change?

Reply:
Nope - this was some experimental code that should be removed. I pushed up a new commit that reverted it back to CJS.